flink-autoscaler/src/test/java/org/apache/flink/autoscaler/RestApiMetricsCollectorTest.java

```diff
@@ -44,18 +44,21 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.Timeout;
 
+import java.time.Duration;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.CompletableFuture;
 import java.util.stream.Collectors;
 
+import static org.assertj.core.api.Assertions.assertThat;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.fail;
+import static org.testcontainers.shaded.org.awaitility.Awaitility.await;
```

Comment:
I think we should add the awaitility lib to the flink-autoscaler as well and not use shaded classes. The `awaitility.version` property seems to be duplicated already, can you please also remove it from `flink-kubernetes-operator-api` POM and move it to parent?

Reply:
Fixed in https://github.com/apache/flink-kubernetes-operator/pull/896/commits/f95263ce7fae4fd1fe953ac2b401b278245914d0 and I've bumped it to the latest release in https://github.com/apache/flink-kubernetes-operator/pull/896/commits/c00d71faee39619f4b1b33a10020c71f0cb88cce

Reply:
Whoops, fixed with 19f25e5ef61a86d5fb36fd52a58ea71167f7f451